modules/eks_nodegroup/launch_templates_spot.tf

```diff
@@ -47,6 +47,7 @@ resource "aws_launch_template" "lt_spot" {
         "Name"          = "${module.generator.prefix}-${local.get_nodegroup_postfix}-spot",
         "Type"          = "spot",
         "nodeGroupName" = local.get_nodegroup_postfix
+        "assign_pods_to_key_nodes" = var.assign_pods_to_key_nodes
```

Comment:
could I ask you to make fmt code before apply it?
terrafrom fmt in the folder

Reply:
Was applied fmt format.